boards/arm/mec1501modular_assy6885/mec1501modular_assy6885_defconfig

```diff
@@ -17,5 +17,6 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_PINMUX=y
```

Comment:
could you please remove `CONFIG_PINMUX=y`

Reply:
pinmux is not yet completely removed, can be removed only at last once all drivers stop using pinmux, this PR is still work in progress

Reply:
I see, sorry my bad.